### PR TITLE
Package auth android dlls

### PR DESCRIPTION
**Pull request by guperrot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the UWP implementation?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Package auth android dlls.

## Related PRs or issues

[AB#60473](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/60473)

### CODE REVIEW

**Comment by guperrot** [comment]
Downloads failing randomly on CI.

**Comment by guperrot** [comment]
Upgrading CI from mono 5.4 to mono 5.12

**Comment by guperrot** [comment]
Screen Shot 2019-04-19 at 9 25 45 PM

File: SDK/AppCenterAuth/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common.csproj [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <RootNamespace>Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common</RootNamespace>
     <AssemblyName>Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common</AssemblyName>
     <FileAlignment>512</FileAlignment>
-    <TargetFrameworkVersion>v8.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v8.0</TargetFrameworkVersion>
```
Comment (MatkovIvan) [inline SDK/AppCenterAuth/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common.csproj]: Is it still the case?

Comment (guperrot) [reply SDK/AppCenterAuth/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common.csproj]: At first build attempt with mono 5.4 yes.
Then I upgraded CI to mono 5.12 only later.
It does not matter though, I kept it like that to align with existing modules.

**Pull request merged**